composer.json

```diff
@@ -111,6 +111,12 @@
         ],
         "post-autoload-dump": [
             "@link-extension"
+        ],
+        "render-docs": [
+            "docker-compose run --rm t3docmake"
```

Comment:
Please let's put the scripts in a namespace so they don't end up hidden in the big list of general Composer command, e.g., `typo3:render-docs` and `typo3:serve-docs`.